src/components/events/partials/ModalTabsAndPages/EventDetailsAssetCatalogDetails.tsx

```diff
@@ -119,7 +120,9 @@ const EventDetailsAssetCatalogDetails = () => {
 										}
 									</td>
 									<td>
-										<a className="fa fa-external-link" href={catalog.url} target="_blank" rel="noreferrer"/>
+										<a href={catalog.url} target="_blank" rel="noreferrer">
+											<LuExternalLink style={{ fontSize: "20px", verticalAlign: "middle" }}/>
+										</a>
```

Comment:
Despite the `verticalAlign` this is vertically off-center. I suspect this applies to all places where you use `LuExternalLink` with these styles.

You could do this instead:
```suggestion
									<a href={catalog.url} target="_blank" rel="noreferrer" style={{ display: "flex"}}>
											<LuExternalLink style={{ fontSize: "20px" }}/>
										</a>